target_chains/ethereum/entropy_sdk/solidity/MockEntropy.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity ^0.8.0;
+
+import {IEntropyConsumer} from "./IEntropyConsumer.sol";
+import "./EntropyErrors.sol";
+
+contract MockEntropy {
```

Comment:
I think this should inherit from `IEntropy`

Reply:
(it's ok to throw an error saying "not implemented" for methods like `register` which are targeted toward providers though)